aips/aip-116.md

```diff
@@ -91,8 +83,8 @@ The new standard introduces new interfaces that conform to [CAIP-122](https://ch
 ### Definitions
 
 - _binding_: The process of adding or modifying fields to fit requirements, sourcing fields from a trusted data source such as the browser window. This process will typically be done by the wallet.
-- _SIWA Message_: A plain text message that is constructed from the `AptosSignInInput` fields by the wallet. This message should follow the format specified in the [Message Construction](#message-construction) section.
-- _SIWA Signing Message_: Bytes output from the [Domain Separated Signing Algorithm](#domain-separated-signing-algorithm) using the SIWA message as input.
+- _SIWA message_: A plain text message that is constructed from the `AptosSignInInput` alongside bound fields by the wallet. This message should follow the format specified in the [Message Construction](#siwa-message-construction) section.
+- _SIWA signing message_: Bytes output from the [Domain Separated Signing Algorithm](#domain-separated-signing-algorithm) using the SIWA message as input.
 - _Signing Request_: A request from an application to the wallet to conduct a certain operation such as signing a message.
 
 ### AptosSignInInput
```

Comment:
What I would like this to explain is what does it mean for a field to be "mandatory".

AFAICT, it means that the dapp has to fill it in before sending the signing request to the wallet

Reply:
Bump.